syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py

```diff
@@ -64,6 +64,24 @@ def download(target_path: Path, version: str):
         logging.info(f"File {target_file} found, skipping download.")
 
 
+_FCNET_SEED_KEY = "replication"
+
+
+def _check_whether_fcnet(benchmark: BenchmarkSet) -> bool:
+    return benchmark.config.config_id == "fcnet"
+
+
+def _check_whether_iaml(benchmark: BenchmarkSet) -> bool:
+    return benchmark.config.config_id.startswith("iaml_")
+
+
+def _check_whether_rbv2(benchmark: BenchmarkSet) -> bool:
+    return benchmark.config.config_id.startswith("rbv2_")
```

Comment:
Those seem unused, if so could you remove it?

Reply:
Yes, they are only needed in the next PR, so I'll remove them here.